tq/transfer_queue.go

```diff
@@ -763,6 +774,17 @@ func (q *TransferQueue) toAdapterCfg(e lfsapi.Endpoint) AdapterConfig {
 	}
 }
 
+var (
+	// contentTypeWarning is the message printed when a server returns an
+	// HTTP 422 at the end of a push.
+	contentTypeWarning = []string{
+		"Uploading failed due to unsupported Content-Type header(s).",
```

Comment:
This error message is great! I think this will help users a lot if they run into this error

Reply:
Great! Thanks